.travis.yml

```diff
@@ -16,6 +18,13 @@ before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y liboctave-dev
   - sudo apt-get install -y octave
+  - if [ "x$OCT_DEV" = "xyes" ]; then
+        sudo apt-get install -y libarpack2-dev libgl2ps-dev libgraphicsmagick++-dev libqrupdate-dev libsuitesparse-dev;
+        wget https://s3.amazonaws.com/octave-snapshot/public/octave-ubuntu-trusty-snapshot.tar.xz;
+        sudo tar --extract --directory=/usr/local --strip-components=1 --file=octave-ubuntu-trusty-snapshot.tar.xz;
+        sudo chmod -R a+rX /usr/local/;
```

Comment:
This was a hack, I've fixed it in my scripts for building the image, shouldn't be needed after the next build